### ISSUE TITLE
[LayoutLM] How to use the text_predictions.txt

### ISSUE THREAD

**Issue by AHardReset** [microsoft/unilm#161]
With the FUNDS fine-tuning model.

With the image:
<details close>
<summary>82250337_0338.png (click to expand)</summary>

![82250337_0338](https://user-images.githubusercontent.com/22583153/83318505-770d4900-a1ea-11ea-86f2-229473d33d59.png)

</details>

_in testing_data FUNDS dataset_

<details close>
<summary>the result for --do_predict is: (click to expand)</summary>

```
TO: S-QUESTION
FROM: S-QUESTION
DATE: S-QUESTION
MANUFACTURER: S-QUESTION
BRAND: S-QUESTION
Oct. S-QUESTION
Dec. S-QUESTION
Jan. S-QUESTION
Nov. S-QUESTION
X S-ANSWER
B&W S-QUESTION
82250337 O
COMPETITIVE B-HEADER
PRODUCT I-HEADER
INTRODUCTION I-HEADER
PROGRESS I-HEADER
REPORT E-HEADER
Sam B-ANSWER
Zolot E-ANSWER
Kool B-ANSWER
Waterfall E-ANSWER
All I-ANSWER
Packings E-ANSWER
TYPE B-QUESTION
OF I-QUESTION
PACKINGS: E-QUESTION
D. B-ANSWER
J. I-ANSWER
Landro E-ANSWER
2- B-ANSWER
Dec- I-ANSWER
97 E-ANSWER
REPORTING B-QUESTION
PERIODS: E-QUESTION
TEST B-QUESTION
MARKET I-QUESTION
GEOGRAPHY: E-QUESTION
Divisions B-ANSWER
621 I-ANSWER
and I-ANSWER
627 I-ANSWER
wisconsin E-ANSWER
PRICE B-QUESTION
POINT: E-QUESTION
FULLS I-QUESTION
$ I-QUESTION
P/ I-QUESTION
V I-QUESTION
$ E-QUESTION
(Indicate I-QUESTION
Distributor's I-QUESTION
Cost I-QUESTION
Per I-QUESTION
Carton) E-QUESTION
SALES B-HEADER
FORCE I-QUESTION
INVOLVEMENT: E-HEADER
They B-ANSWER
have I-ANSWER
crew- I-ANSWER
worked I-ANSWER
distribution, I-ANSWER
and I-ANSWER
it I-ANSWER
is I-ANSWER
reported I-ANSWER
that I-ANSWER
they I-ANSWER
may I-ANSWER
crew- I-ANSWER
work I-ANSWER
it I-ANSWER
again I-ANSWER
Sales I-ANSWER
force I-ANSWER
has I-ANSWER
been I-ANSWER
busy E-ANSWER
promoting I-ANSWER
old I-ANSWER
style I-ANSWER
backs I-ANSWER
to I-ANSWER
clean E-ANSWER

```
</details>

How to know the relationship between the output labels or link the answers with his questions?

For example:
TO: S-QUESTION
has the relation:
Sam B-ANSWER
Zolot E-ANSWER

A formated output could be:
`{ "TO":  ["Sam", "Zolot"] }`

Or can be more complex:
Oct. S-QUESTION
Dec. S-QUESTION
Jan. S-QUESTION
Nov. S-QUESTION
X S-ANSWER

when X S-ANSWER only belongs to Nov. S-QUESTION
and the others does not have a result (blank in form, reffer to the image).

Can somebody help me please? :)

**Comment by wolfshow** [comment]
@AHardReset You may follow the FUNSD paper, which introduces an approach to link the key-value pair by another binary classifier.

**Comment by SandyRSK** [comment]
Hi,
How to use the pretrained layoutlm model directly to our task such as receipt understanding.
Is the pretrained model able to predict the receipt information.?
Thank you

**Comment by yuanhuang0825** [comment]
＠walfshow  so text_pridictioons.txt isn't a linked output .need use another classifier to link it?

**Comment by kbrajwani** [comment]
@wolfshow  i am wondering how binary classifier is used for linking the question-answer. can you explain a bit.
